src/main/java/de/themoep/inventorygui/InventoryGui.java

```diff
@@ -144,12 +144,17 @@ public class InventoryGui implements Listener {
             } catch (IllegalArgumentException | IncompatibleClassChangeError ignored) {}
         }
         if (clickSound == null) {
-            for (Sound sound : Sound.values()) {
-                if (sound.name().contains("CLICK")) {
-                    // Convert to sound key under the assumption that the enum name is just using underscores in the place of dots
-                    clickSound = sound.name().toLowerCase(Locale.ROOT).replace('_', '.');
-                    break;
+            try {
+                Sound[] sounds = (Sound[]) Sound.class.getDeclaredMethod("values").invoke(null);
+                for (Sound sound : sounds) {
+                    if (sound.name().contains("CLICK")) {
+                        // Convert to sound key under the assumption that the enum name is just using underscores in the place of dots
+                        clickSound = sound.name().toLowerCase(Locale.ROOT).replace('_', '.');
+                        break;
+                    }
                 }
+            } catch (InvocationTargetException | IllegalAccessException | NoSuchMethodException e) {
+                e.printStackTrace();
```

Comment:
Please use the plugin's logger to log this, otherwise there will be a huge warning and error on modern servers. Ideally at the `FINE` level as I don't think it needs to be something that the normal users cares about, just for debugging is fine.

Reply:
Ah, right. Then I would say to simply not log this. It's done like that for the missing sounds further up already anyways.